Model/Push.php

```diff
@@ -1575,6 +1579,57 @@ public function isPayPerEmailB2BModePushPaid()
         return $this->isPayPerEmailB2BModePush();
     }
 
+    private function getOrderIncrementId()
+    {
+        $brqOrderId = false;
+
+        if (isset($this->postData['brq_invoicenumber']) && strlen($this->postData['brq_invoicenumber']) > 0) {
+            $brqOrderId = $this->postData['brq_invoicenumber'];
+        }
+
+        if (isset($this->postData['brq_ordernumber']) && strlen($this->postData['brq_ordernumber']) > 0) {
```

Comment:
Should this statement be allowed to overwrite the previous one? Or is it always one or the other? If the latter is the case, can we add a return in the if statements?